.drone.yml

```diff
@@ -4618,15 +4630,20 @@ steps:
         path: /rpmrepo
     commands:
     - mkdir -p /rpmrepo/teleport/cache
-    # we explicitly want to delete anything present locally which has been deleted
-    # from the upstream S3 bucket
+    # Explicitly delete anything present locally before copying over new assets
+    # to ensure the repository remains in sync.
     - aws s3 sync s3://$AWS_S3_BUCKET/teleport/ /rpmrepo/teleport/ --delete
     - mkdir -p /rpmrepo/teleport/${DRONE_TAG##v}
-    - cp -a /go/artifacts/*.rpm /rpmrepo/teleport/${DRONE_TAG##v}/
+    # This is a hack for Teleport 8. Copy over CentOS 7 RPMs instead of the
+    # mainline RPMs which only run on CentOS 8+.
+    #
+    # In Teleport 9 we will switch the buildbox to CentOS 7 and no longer need
+    # to do this. This should never make it into master or branch/v9.
+    - cp -a /go/artifacts/*centos7*.rpm /rpmrepo/teleport/${DRONE_TAG##v}/
```

Comment:
@webvictim I'm pretty sure this will work, because if I look at the metadata for the RPM it shows up correctly even though the filenames contain CentOS 7 in them. What do you think?

Do we have any way to test this by the way?

```
$ rpm -qip teleport-8.0.6-1-centos7.x86_64.rpm
Name        : teleport
Version     : 8.0.6
Release     : 1
Architecture: x86_64
Install Date: (not installed)
Group       : default
Size        : 210786031
License     : Apache-2.0
Signature   : (none)
Source RPM  : teleport-8.0.6-1.src.rpm
Build Date  : Fri Dec 17 00:14:49 2021
Build Host  : 0bb7e33aa24a
Relocations : / 
Packager    : info@goteleport.com
Vendor      : Gravitational
URL         : https://goteleport.com/docs
Summary     : Teleport is a gateway for managing access to clusters of Linux servers via SSH or the Kubernetes API [64-bit Open source edition]
Description :
Teleport is a gateway for managing access to clusters of Linux servers via SSH or the Kubernetes API [64-bit Open source edition]
```